cmd/config.go

```diff
@@ -73,6 +74,7 @@ func newConfigCommand(config *settings.Config) *cobra.Command {
 	}
 	validateCommand.Annotations["<path>"] = configAnnotations["<path>"]
 	validateCommand.PersistentFlags().StringVarP(&configPath, "config", "c", ".circleci/config.yml", "path to config file")
+	validateCommand.PersistentFlags().BoolVarP(&quietOutput, "quiet", "q", false, "Suppress verbose output")
```

Comment:
happy to invert this, in fact, I'd prefer it that way, like `-v` to enable this output, and default is suppress it.

Reply:
I'm currently working through some migration work for these commands and would prefer if this was inverted to the more common `-v` verbose approach.

Reply:
@elliotforbes 👍 updated

Reply:
Much appreciated! 🙌🏼

Reply:
No worries - this was my preference anyway, but tried to do it in the least obtrusive way to start with.